Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -1298,6 +1363,9 @@ def main() -> None:
         elif command == "reco-change-alert-status":
             result = change_alert_status(reco_client, demisto.args()["alert_id"], demisto.args()["status"])
             return_results(result)
+        elif command == "reco-get-user-context-by-email-address":
+            result = get_user_context_by_email_address(reco_client, demisto.args()["email_address"])
```

Comment:
Please generate a README.md file for your integration.
To do that, please run the `demisto-sdk generate-docs` command (use the -e flag for using a commad_examples file or -c for generating a specific command) - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#generate-docs.